cpp/tests/io/parquet_v2_test.cpp

```diff
@@ -688,60 +687,9 @@ TEST_P(ParquetV2Test, PartitionedWriteEmptyColumns)
   CUDF_TEST_EXPECT_TABLES_EQUAL(expected2, result2.tbl->view());
 }
 
-TEST_P(ParquetV2Test, LargeColumnIndex)
```

Comment:
Can you help me understand why this test has been removed?

Reply:
That's a great question, I should have left a comment before reviews.
This test was included only to test the case where the writer writes the data in two batches (not the same as chunks!). Batching has since been disabled so we don't need this (huge) test.

Reply:
Understood, thanks!